content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -95,9 +95,9 @@ The first step when using Prisma Client is installing the `@prisma/client` npm p
 npm install @prisma/client
 ```
 
-Installing the `@prisma/client` package invokes the `prisma generate` command, which reads your Prisma schema and _generates_ the Prisma Client code. The code will be located in `node_modules/@prisma/client`.
+Installing the `@prisma/client` package invokes the `prisma generate` command, which reads your Prisma schema and _generates_ the Prisma Client code. The code will be located in `node_modules/.prisma/client`.
```

Comment:
"The code will be located ..." should probably link to where we explain how this works in detail.

Reply:
Added link to https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#the-prismaclient-npm-package